### ISSUE TITLE
add class parameter to sc_cookbook_recipe_thumb()

### ISSUE THREAD

**Issue by Jimmi08** [Moc/cookbook#22]
## Motivation

Your output is
`'<img class="img-responsive" src="'.$thumbUrl.'" alt="'.$this->sc_cookbook_recipe_name('no_url').'" />';`

Image class should be parameter as in other part of core

 Thanks

**Comment by Jimmi08** [comment]
This is alternative but it is very non standard solution:

```
 {SETIMAGE: w=0&h=0}
<img class="card-img-top" src="{COOKBOOK_RECIPE_THUMB=url}" alt="{COOKBOOK_RECIPE_NAME=no_url}">
```

**Comment by Moc** [comment]
Fixed in #21 and https://github.com/Moc/cookbook/commit/a0783d6fcbb7a60faa4afc8962fec9d74f28a972
